.travis.yml

```diff
@@ -181,6 +181,7 @@ env:
     # - "TESTDIR=Rust/iron"
     # - "TESTDIR=Rust/nickel"
     # - "TESTDIR=Rust/hyper"
+    # - "TESTDIR=Rust/may-minihttp"
```

Comment:
This line should NOT be commented. There is a revert of some accidental merge of the Docker branch into master (https://github.com/TechEmpower/FrameworkBenchmarks/pull/3357) several commits later.
@Xudong-Huang @nbrady-techempower

Reply:
This is in #3358 now

Reply:
Hope someone would fix it, I just see all rust related test is disabled currently, I think there is a reason.

Reply:
@Xudong-Huang it is fixed in the docker branch. We're moving toward docker for all test implementations.